### DISCUSSION TITLE
When will the RUST version of @rspress/plugin-preview be online?

### DISCUSSION THREAD

**Discussion by peterczg** [web-infra-dev/rspress#1658] [Q&A]
I‘m just wondering, because my team is using dumi now, we have a lot of components that need to show previews, and also plenty of markdowns, the current compilation performance of js plug-in is not ideal enough, we are looking forward to the rust compiler!

**Comment by Timeless0911** [comment]
We're planning to initiate the design and development of Rspress 2.0 in the first half of 2025 (Q1-Q2). As part of this, we aim to refactor and unify the preview and playground plugins. The current implementation within mdx-rs presents limitations, and we'll be mindful of this in future implementations @SoonIter . We can't pinpoint an exact timeline for this yet, but we'll be discussing the design of the preview plugin in the issue area. If you're interested, please keep an eye out for those discussions.
